lib/include/jxl/cms_interface.h

```diff
@@ -64,11 +64,19 @@ typedef struct {
  * @param output_profile the colorspace to which JxlCmsInterface::run should
  *        convert the input data.
  * @param intensity_target for colorspaces where luminance is relative
- *        (essentially: not PQ), indicates the luminance of (1, 1, 1). This is
- *        only useful for conversions between a relative luminance colorspace
- *        and an absolute luminance one, in either direction.
- *        @p intensity_target cd/m² in the absolute colorspace should map to and
- *        from (1, 1, 1) in the relative one.
+ *        (essentially: not PQ), indicates the luminance at which (1, 1, 1) will
+ *        be displayed. This is useful for conversions between PQ and a relative
+ *        luminance colorspace, in either direction: @p intensity_target cd/m²
+ *        in PQ should map to and from (1, 1, 1) in the relative one.\n
```

Comment:
I was checking the generated html documentation to see how this `\n` gets rendered (I don't know if that becomes a line break or a literal "\n"). Looks like all of the documentation in this file doesn't actually get generated though, for some reason. Could you look into that?

Reply:
From what I can tell, it does but the struct gets its own separate page. Is that also what happens for you?

`\n` is equivalent to `@n` which does generate a newline: https://www.doxygen.nl/manual/commands.html#cmdn (Not sure what it has to do with the printf function but oh, well.)

Reply:
Well I'm just looking at the docs generated from this PR by following the link at the bottom of the "Conversation" page: https://libjxl--942.org.readthedocs.build/en/942/
I cannot find any documentation for these functions there. Maybe something has to be added to a higher level index or something?

Reply:
Oh, yes, I was looking at the direct Doxygen output. For Sphinx, it was indeed missing an `@addtogroup`. I have put it in `libjxl_common` given that we would like to eventually also have a high-level decoding API that would use it. How does that sound?

Reply:
OK for me, though maybe it would make sense to have a separate group for all color encoding / color management related things.

In general, regarding the documentation, I think we need a bit more than just documentation of every struct and function. We also need a more high-level overview, and this is especially true for the color encoding / color management stuff since it is likely that the API will be used in wrong ways if we don't carefully explain things (and part of the documentation will have to be basically reader education rather than actual libjxl documentation).

(but that can of course wait for a different PR :))